planning/motion_velocity_smoother/src/motion_velocity_smoother_node.cpp

```diff
@@ -72,6 +73,13 @@ MotionVelocitySmootherNode::MotionVelocitySmootherNode(const rclcpp::NodeOptions
     "~/input/operation_mode_state", 1,
     [this](const OperationModeState::ConstSharedPtr msg) { operation_mode_ = *msg; });
 
+  srv_adjust_common_param = create_service<SetBool>(
+    "~/adjust_common_param", std::bind(&MotionVelocitySmootherNode::onAdjustParam, this, _1, _2));
+  adjustParam = false;
```

Comment:
Is this variable necessary?

Reply:
Thought it might not be necessary to set the force acceleration parameter every time when the request is continuously true. The same applies when false is continuous.
What do you think about it?